### ISSUE TITLE
reloading OSX Safari 7.1 Mavericks after a call using Temasys plugin fails

### ISSUE THREAD

**Issue by JamieMcDonnell** [Temasys/AdapterJS#23]
Hi there,

another one I'm afraid - it seems consistent after ending a call in Safari OSX Mavricks using your plugin, if I try to reload the page to make another call the page fails to reload. I have to kill safari and open it again to be able to reload the page and make another call.

Please let me know if you need any more details in how to reproduce this issue.

Thanks

Jamie

**Comment by johache** [comment]
Alright, I was able to reproduce the issue.
I'll open a new ticket on this.

**Comment by johache** [comment]
Hey Iñaki,

I believe I fixed the issue on my side;
However, when I tried with you application, the issue had disappear with both my locally built plugin and the released one (0.8.810)

Did you change your code base?

**Comment by ibc** [comment]
Yes, I removed the "closeMediaStream(localStream") once the PeerConnection is closed. I've re-enabled it and yes, it freezes Safari.

**Comment by johache** [comment]
FYI, I tested against our current internal build and the issue was gone.
A new version including this fix should be released next week.

**Comment by ibc** [comment]
2015-06-30 11:22 GMT+02:00 Jacques-Olivier Haché <notifications@github.com>:

> FYI, I tested against our current internal build and the issue was gone.
> A new version including this fix should be released next week.
>
Thanks!


-- 
Iñaki Baz Castillo
<ibc@aliax.net>
